pmd-java/src/main/resources/rulesets/java/migrating.xml

```diff
@@ -477,4 +477,32 @@ public class MyTest {
     </example>
   </rule>
 
-</ruleset>
\ No newline at end of file
+    <rule name="ForLoopCanBeForeach"
+          language="java"
+          since="6.0"
+          message="This 'for' loop can be replaced by a 'foreach' loop"
+          class="net.sourceforge.pmd.lang.java.rule.migrating.ForLoopCanBeForeachRule"
+          externalInfoUrl="${pmd.website.baseurl}/pmd_rules_java_migrating.html#forloopcanbeforeach">
```

Comment:
I've seen this now for some other rules: For completeness (it probably has no real usage, only if you happen to work on a code base < java 1.5), we should add `minimumLanguageVersion="1.5"`